src/coreclr/src/gc/gc.cpp

```diff
@@ -38609,6 +38609,7 @@ void gc_heap::walk_heap_per_heap (walk_fn fn, void* context, int gen_number, BOO
     uint8_t*       end = heap_segment_allocated (seg);
     BOOL small_object_segments = TRUE;
```

Comment:
>BOOL small_object_segments = TRUE; [](start = 4, length = 34)

you might want to get rid of the small_object_segments var since you got rid of the usage in the loop below

Reply:
Thanks!